packages/wallet-service/src/txProcessor.ts

```diff
@@ -81,6 +81,8 @@ export const IGNORE_TXS = {
   ],
 };
 
+export const CREATE_NFT_MAX_RETRIES = 5;
```

Comment:
We used to have an env var for this, why did you change it?

```
export const MAX_METADATA_UPDATE_RETRIES: number = parseInt(process.env.MAX_METADATA_UPDATE_RETRIES || '3', 10);
```

Reply:
Refactored it in 4448e29f567e858ebbc5950d461e5ed366ce0289

We were not using this env, it was always defaulting to 3, added it to codebuild as well